ibm-messaging-mq-cloud-showcase-app/frontend/AWS-MQ-showcase/.circleci/config.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
```

Comment:
These look like editor configurations that shouldn't be included in this pull request.